packages/components/src/List/ListComposition/utils.js

```diff
@@ -0,0 +1,5 @@
+export const columnsFromChildrens = children => {
+	return Array.isArray(children)
+		? children.filter(column => column.props?.dataKey).map(column => column.props)
```

Comment:
could the data key be '0' ?

```suggestion
		? children.filter(column => column.props?.dataKey != null).map(column => column.props)
```

Reply:
This code comes from https://github.com/Talend/ui/pull/3842/files#diff-8d68d34558e914cec5b15cc6aac0ffddb5fdacc6885de7eec205647f55f38cc3L26
but we can improve it for sure ! Thank you